src/main/java/org/haffson/adventofcode/days/day01/Day01.java

```diff
@@ -40,100 +39,59 @@ public int getDay() {
     }
 
     @Override
-    public HashMap<String, ProblemStatusEnum> getProblemStatus() {
+    public Map<Integer, ProblemStatusEnum> getProblemStatus() {
         return problemStatus;
     }
 
     @Override
     public String firstPart() {
-        try {
-            return "Product 1: " + calculateNumber1();
-        } catch (FileNotFoundException e) {
-            return "error";
-        }
+        return "Product 1: " + calculateProduct_Part1(numbers);
     }
 
     @Override
     public String secondPart() {
-        try {
-            return "Product 2: " + calculateNumber2();
-        } catch (FileNotFoundException e) {
-            return "error";
-        }
+        return "Product 2: " + calculateProduct_Part2(numbers);
+    }
+
+
+    // utility method: subtract numbers by 2020
```

Comment:
JavaDoc. Außerdem: Du substractest nicht by, sondern from.

```suggestion
    /**
     * Utility method that subtracts each number from 2020.
     */
```